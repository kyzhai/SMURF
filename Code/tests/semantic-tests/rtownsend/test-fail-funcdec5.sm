/* Pattern lists must be unique for multiple function declarations */
f :: [Int] -> Int -> Int

f [0,1,2] a = 4
f [0,1,2] m = 6

main = []
