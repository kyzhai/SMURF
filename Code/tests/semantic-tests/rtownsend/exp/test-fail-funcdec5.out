Fatal error: exception Sast.Multiple_identical_pattern_lists("[0, 1, 2] m")
